R/classicProcess.R

```diff
@@ -1143,7 +1153,7 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
   ))
 
   if (inherits(fittedModel, "try-error")) {
-    return(gettextf("Estimation failed\nMessage:\n%s", attr(fittedModel, "condition")$message))
+    return(gettextf("Estimation failed: %s", gsub("lavaan ERROR:", "", attr(fittedModel, "condition")$message)))
   }
```

Comment:
this is just a minor point, but you *could* use `jaspBase::isTryError` and `jaspBase::.extractErrorMessage`